src/__tests__/model.test.ts

```diff
@@ -1240,6 +1528,32 @@ describe('model', () => {
       expectType<Date>(result.updatedAt);
     });
 
+    test('timestamp config schema', async () => {
```

Comment:
We need another test case for `insertOne()` with the `_createdDate` and `_updatedDate` values populated.